### PR TITLE
Fixing typos - Part 1

### PR DESCRIPTION
**Pull request by Fourchaux**
**Kind:** documentation.

Note 1: Typos found with https://github.com/codespell-project/codespell

Note 2: Three parts are planned.

Note 3: I did not replace the few terms "_focussed_" by "_focused_". 
These replacements seem preferable. Can I do them?

### CODE REVIEW

**Comment by Zimmi48** [comment]
Given what I read here https://en.wiktionary.org/wiki/focussed, I think that moving from "focussed" to "focused" would be OK. @herbelin Do you agree?

**Review by Zimmi48 — APPROVED** [review]
Thanks!

**Comment by Zimmi48** [comment]
@vbgl What milestone do you want to put this in?

**Comment by herbelin** [comment]
>  @herbelin Do you agree?

Didn't we have already a discussion on focused vs focussed with a native American speaker? My vague memories are that it does not matter. But, one "s" seems indeed more common.

File: dev/doc/build-system.dev.txt [hunk 1]
```diff
@@ -87,7 +87,7 @@ Cons:
    clear-text generated .ml.
 
 
-Makefiles hierachy
+Makefiles hierarchy
 ------------------
```
Comment (cpitclaudel) [inline dev/doc/build-system.dev.txt]: I think this might cause warnings (the underline will be too short)

Comment (SkySkimmer) [reply dev/doc/build-system.dev.txt]: Not in a .txt surely

Comment (cpitclaudel) [reply dev/doc/build-system.dev.txt]: Oh, sorry, I thought this was an rst

**Comment by jfehrle** [comment]
One "s" is most common in the US.

File: dev/doc/build-system.txt [hunk 2]
```diff
@@ -18,8 +18,8 @@ See http://www.gnu.org/software/make/manual/make.htmlPrerequisite-Types
 * Annotation before commands: +/-/@
 
 a command starting by - is always successful (errors are ignored)
-a command starting by + is runned even if option -n is given to make
-a command starting by @ is not echoed before being runned
+a command starting by + is ran even if option -n is given to make
```
Comment (jfehrle) [inline dev/doc/build-system.txt]:
```suggestion
a command starting by + is run even if option -n is given to make
```

Shouldn't be in the past tense in the original or the fix.

File: dev/doc/build-system.txt [hunk 3]
```diff
@@ -18,8 +18,8 @@ See http://www.gnu.org/software/make/manual/make.htmlPrerequisite-Types
...
+a command starting by @ is not echoed before being ran
```
Comment (jfehrle) [inline dev/doc/build-system.txt]:
```suggestion
a command starting by @ is not echoed before being run
```

**Review by jfehrle — CHANGES_REQUESTED** [review]
Looks good.  2 more small changes.

File: dev/doc/build-system.txt (hunk 3 above)
Comment (Fourchaux) [reply dev/doc/build-system.txt]: OK. s/ran/run/

**Review by jfehrle — APPROVED** [review]
Looks good.  Nice to clean up the typos.

**Review by gares — APPROVED** [review]
Thanks

**Review by vbgl — APPROVED** [review]
Nice job.

**Comment by Zimmi48** [comment]
> Didn't we have already a discussion on focused vs focussed

That's what I recalled, and that's why I asked you specifically.

**Comment by Zimmi48** [comment]
@Fourchaux As a general advice, not just for contributing to Coq, but for opening PRs on any project, I recommend that you always create specific "topic" branches, instead of using your master branch. `master` should always track upstream.

**Pull request merged**